Security/Authentication/Provider/JWTProvider.php

```diff
@@ -117,4 +122,16 @@ public function setUserIdentityField($userIdentityField)
     {
         $this->userIdentityField = $userIdentityField;
     }
+
+    /**
+     * @param JWTDecodeFailureException $previous
+     *
+     * @return AuthenticationException
+     */
+    private function createInvalidJWTException(JWTDecodeFailureException $previous = null, $message = 'Invalid JWT Token')
```

Comment:
Small thing, but we *never* pass the `$message` arg - so we could just *not* have this be an argument at all.